engine/client/cl_view.c

```diff
@@ -286,9 +286,6 @@ qboolean V_PreRender( void )
 	if( !ref.initialized )
 		return false;
 
-	if( host.status == HOST_NOFOCUS )
```

Comment:
What's the reason of this change?

Reply:
Guess it came from Velaron's fork

Reply:
IIRC, it was causing sound issues when the game window is inactive (not minimized) as I was testing the voice chat basically with myself on the same machine

Reply:
@Velaron I don't remember the actual desired behavior in this case.

I think for the sake of debugging, it's better to limit fps and mute only in minimized state. But then NOFOCUS state make no sense.

Reply:
What do with this? Keep or remove?

Reply:
It's ok.